src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/HttpApi/ApiClient.cs

```diff
@@ -304,10 +304,53 @@ public async Task<string> GetMetricsAsync(CancellationToken token)
+        {
+            using HttpRequestMessage request = new(HttpMethod.Get, routeAndQuery);
+            HttpResponseMessage response = await SendAndLogAsync(request, HttpCompletionOption.ResponseHeadersRead, token).ConfigureAwait(false);
+            return response;
+        }
+
+        public async Task<OperationResponse> EgressTraceAsync(int processId, int durationSeconds, string egressProvider, CancellationToken token)
+        {
+            string uri = FormattableString.Invariant($"/trace?pid={processId}&egressProvider={egressProvider}&durationSeconds={durationSeconds}");
+            using HttpRequestMessage request = new(HttpMethod.Get, uri);
+            using HttpResponseMessage response = await SendAndLogAsync(request, HttpCompletionOption.ResponseContentRead, token).ConfigureAwait(false);
+
+            return new OperationResponse(response.StatusCode, response.Headers.Location, await response.Content.ReadAsStringAsync().ConfigureAwait(false));
+        }
+
+        public async Task<List<Models.OperationSummary>> GetOperations(CancellationToken token)
+        {
+            using HttpRequestMessage request = new(HttpMethod.Get, "/operation");
+            using HttpResponseMessage response = await SendAndLogAsync(request, HttpCompletionOption.ResponseContentRead, token).ConfigureAwait(false);
+            response.EnsureSuccessStatusCode();
+
+            return await ReadContentEnumerableAsync<Models.OperationSummary>(response).ConfigureAwait(false);
+        }
+
+        public async Task<OperationStatus> GetOperationStatus(Uri operation, CancellationToken token)
+        {
+            using HttpRequestMessage request = new(HttpMethod.Get, operation.ToString());
+            using HttpResponseMessage response = await SendAndLogAsync(request, HttpCompletionOption.ResponseContentRead, token).ConfigureAwait(false);
+            response.EnsureSuccessStatusCode();
+
+            return await ReadContentAsync<OperationStatus>(response).ConfigureAwait(false);
+        }
+
+        public async Task<HttpStatusCode> CancelEgressOperation(Uri operation, CancellationToken token)
+        {
+            using HttpRequestMessage request = new(HttpMethod.Delete, operation.ToString());
+            using HttpResponseMessage response = await SendAndLogAsync(request, HttpCompletionOption.ResponseContentRead, token).ConfigureAwait(false);
+            response.EnsureSuccessStatusCode();
+
+            return response.StatusCode;
+        }
+
         private static async Task<T> ReadContentAsync<T>(HttpResponseMessage responseMessage)
         {
             using Stream contentStream = await responseMessage.Content.ReadAsStreamAsync().ConfigureAwait(false);
-            return await JsonSerializer.DeserializeAsync<T>(contentStream).ConfigureAwait(false);
+            return await JsonSerializer.DeserializeAsync<T>(contentStream, new JsonSerializerOptions { PropertyNameCaseInsensitive = true }).ConfigureAwait(false);
```

Comment:
Why is this necessary? The models used by the tests should be exactly the same as used by the product, so there should be no disagreement of casing.

Reply:
I will annotate the model to reflect the default serialization style instead of doing this.